src/app/components/TabBar/index.tsx

```diff
@@ -0,0 +1,69 @@
+import Routes from '@constants/routes';
+import React, { useEffect, useState } from 'react';
+import { View, ImageSourcePropType, Image } from 'react-native';
+import IC_HOME_INACTIVE from '@assets/TabBar/ic_home_inactive.png';
```

Comment:
in projects we usually use camel and lower to name files and variables

Reply:
changed